### PR TITLE
simplify findHigherSheet in dom renderer

### PR DESCRIPTION
**Pull request by behnammodi**
Used `find` instead of `for` to simplify and reduce the size

### CODE REVIEW

File: packages/jss/src/DomRenderer.js [hunk 1]
```diff
@@ -130,17 +130,14 @@ const getHead = memoize((): HTMLElement => (document.querySelector('head'): any)
...
-    if (
-      sheet.attached &&
-      sheet.options.index > options.index &&
-      sheet.options.insertionPoint === options.insertionPoint
-    ) {
-      return sheet
-    }
-  }
-  return null
+  return (
```
Comment (kof) [inline packages/jss/src/DomRenderer.js:133]: this is mostly a cosmetic change that is also slower (on a micro level), we optimize everything in this lib to get the smallest bit of performance.

Comment (behnammodi) [reply packages/jss/src/DomRenderer.js:133]: So you mean this PR is not a good effect on JSS?

Comment (behnammodi) [reply packages/jss/src/DomRenderer.js:133]: I agree `for` is faster than `find`

Comment (kof) [reply packages/jss/src/DomRenderer.js:133]: thank you

**Pull request closed without merging**